gen/calc.py

```diff
@@ -658,6 +658,14 @@ def validate_exhibitor_storage_master_discovery(master_discovery, exhibitor_stor
             "`master_http_load_balancer` then exhibitor_storage_backend must not be static."
 
 
+def validate_adminrouter_grpc_proxy_port(adminrouter_grpc_proxy_port):
+    try:
+        assert 0 < int(adminrouter_grpc_proxy_port) < 65536
```

Comment:
Should we exclude ports 80 and 443 as well?

Reply:
I'm 50/50 on restricting it down, on the one hand most peeps wouldn't want it on those ports, but if for whatever reason they do want it on one of those, then they'd have to hack it up.

If you have a strong opinion to restrict it, let me know and I'll add it, otherwise I'd say lets just let it ride ;).

Reply:
Admin Router already uses ports 80 and 443 and this is hardcoded [1] If somebody tries to have gRPC port on port 80/443 then the things will break.

Should we be worried about it?

[1] https://github.com/mesosphere/dcos-enterprise/blob/78fc339ba732111331e1ae08d8c106a3632b85c6/gen_extra/dcos-config.yaml#L8-L28

Reply:
We have many other ports in this range that will not work. I'm happy to keep it as-is.

Reply:
I also think that it's not going to be easy to figure out dynamically _what_ to exclude. I am also fine to keep it as-is.